src/main/java/spoon/support/visitor/java/reflect/RtMethod.java

```diff
@@ -138,9 +138,8 @@ public int hashCode() {
 	}
 
 	/**
-	 * Creates a new {@link RtMethod}
-	 * @param method is passed as Method
-	 * @return {@link RtMethod}
+	 * @param method is the Method which is to be wrapped in a RtMethod
+	 * @return an object of RtMethod after calling the parameterized constructor of the RtMethod class
```

Comment:
The `@param` is good in terms of information, but the comment about types still applies. If you want to refer to types (which isn't unreasonable here; this method just wraps an object of one type in an object of another type), then use `@link`.

The `@return` is much too specific. Only write what's actually important to a caller of the method. Does it matter _how_ the wrapper is created? What if we change `RtMethod` to a fluent API instead, should that matter to this method? I would suggest something like `@return a wrapper around the passed method` instead, because that describes the semantics of the value, and not the details of its type or how it came to be.

Reply:
Thanks for the suggestions, I will incorporate the necessary changes in the next commit.